presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/EnvironmentExtender.java

```diff
@@ -15,7 +15,20 @@
 
 import io.prestosql.tests.product.launcher.env.Environment;
 
+import java.util.List;
+
 public interface EnvironmentExtender
 {
-    void extendEnvironment(Environment.Builder builder);
+    Environment.Builder extendEnvironment(Environment.Builder builder);
+
+    static Environment.Builder compose(Environment.Builder builder, List<EnvironmentExtender> extenders)
```

Comment:
I would rather expect the method to take list of extenders and return an extender